src/ergw_config.erl

```diff
@@ -24,6 +24,7 @@
 -endif.
 
 -define(DefaultOptions, [{plmn_id, {<<"001">>, <<"01">>}},
+			 {node_id, undefined},
```

Comment:
Looks like for fix CT errors:
```
sbi_nbsf_SUITE ==> {thrown,{error,{options,{node_id,undefined}}}}
tdf_SUITE ==> {thrown,{error,{options,{node_id,undefined}}}}
```
need to change default value from `undefined` to `<<>>`